<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>512</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>491</width>
    <height>512</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>491</width>
    <height>512</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>HashCrack</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images.qrc">
    <normaloff>:/Images/iconka.png</normaloff>:/Images/iconka.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>431</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Słownik:</string>
    </property>
    <widget class="QLineEdit" name="slownikEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>C:/Users/Patryk/Desktop/slownik.txt</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Otwórz słownik...</string>
     </property>
     <property name="icon">
      <iconset resource="Images.qrc">
       <normaloff>:/Images/open-file.png</normaloff>:/Images/open-file.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>431</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Szukany hash:</string>
    </property>
    <widget class="QLineEdit" name="hashEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>73c18c59a39b18382081ec00bb456d43</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Hash:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Typ hashu:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <item>
      <property name="text">
       <string>MD4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MD5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SHA-1</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>431</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Logi:</string>
    </property>
    <widget class="QPlainTextEdit" name="logiEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>411</width>
       <height>71</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>440</y>
      <width>175</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Odtwórz dźwiek po zakończeniu</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>341</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Images.qrc">:/Images/logo.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
